mailme.go

```diff
@@ -111,20 +112,20 @@ func (t *TemplateCache) Set(key, value string, expirationTime time.Duration) (*t
 }
 
 func (t *TemplateCache) fetchTemplate(url string, triesLeft int) (string, error) {
-	client := http.Client{
+	client := &http.Client{
 		Transport: &http.Transport{
```

Comment:
no need to set the transport. it's going to be replaced in the safehttp call anyways

Reply:
The original transport gets wrapped by the new one. It's stored as `noLocalTransport.inner`.

However, Go's `DefaultTransport` is identical to the one defined here, except that the default has `ForceAttemptHTTP2: true`. Any idea if that's deliberate? If it's not, we can just use the `DefaultTransport`. If it is, I think we should still start with the default and just update that one setting to make it explicit.

Reply:
agree on using `DefaultTransport` 👍